draft-kcs-tvr-requirements.xml

```diff
@@ -333,6 +333,72 @@ Likewise, the assigning of an address property at one layer does not imply the p
       <t>TBD</t>
     </section>
 
+     <section anchor="requirements-summary">
+      <name>Requirements Summary</name>
+
+      <section title="Support the Identification and Advertisement of Node Attribute Changes">
+        <t>In Time-Variant Routing, scheduling of the system are as expected.
+        In practical situations, however, the attributes of nodes can be
```

Comment:
To be consistent with Section 2 these should use the term "property" and "property value" to describe what is changing.